### ISSUE TITLE
Old contact info in the docs

### ISSUE THREAD

**Issue by RollingStar** [beetbox/beets#5462]
### Problem

Looks like the docs still has some traces of old communities that we no longer support.

- mailing lists
- twitter
- https://discourse.beets.io/ (people are still participating there; I get not wanting to use github but it splinters the already microscopic community)
- IRC

I see the windows install mentions the mailing list: https://github.com/beetbox/beets/blob/master/docs/guides/main.rst

and mastodon:

> Please let us know what you think of beets via [the discussion board](https://github.com/beetbox/beets/discussions) or [Mastodon](https://fosstodon.org/@beets).

https://beets.io/

> If you still need help, you can drop by the #beets IRC channel on Libera.Chat, drop by [the discussion board](https://github.com/beetbox/beets/discussions/), send email to [the mailing list](https://groups.google.com/group/beets-users), or [file a bug](https://github.com/beetbox/beets/issues) in the issue tracker. Please let us know where you think this documentation can be improved.

Someone should probably do an aggressive grep search or similar through the source tree (at least the docs) and make sure that old references are replaced with our canonical support locations: github discussions, github issues, and apparently mastodon.

### Mastodon

I like Mastodon, but it's a double edged sword: who is going to check posts there to give people support if they need it? But I do see huge benefit in casual show and tell / I love beets posts happening on Mastodon. The whole Masto community is FOSS adjacent and trumpeting how cool beets is could grow our user base and possibly even our dev base. It looks like some of the b33ts account admins are retweeting cool stuff from people, so that's a great use case.

TLDR: Mastodon for showing off, Github for development and discussion.
